gn2/wqflask/views.py

```diff
@@ -1277,6 +1277,29 @@ def display_genewiki_page(symbol: str):
     )
 
 
+@app.route("/genewiki/<int:comment_id>/history")
+def display_wiki_history(comment_id: str):
+    most_recent = {}
+    previous_versions = []
+    try:
+        entries = requests.get(
+            urljoin(
+                GN3_LOCAL_URL,
+                f"/api/metadata/wiki/{comment_id}/history"
+            )
+        )
+        entries.raise_for_status()
+        if entries := entries.json():
+            most_recent, previous_versions = entries[0], entries[1:]
```

Comment:
I wonder if we can have this logic in the template itself, removing the need for duplicating the two sections. Something like:
```
{% for i in range(len(entries) %}
{% if i == 0 % }
Most Recent Version
{% elif i == 1 %}
Other versions
{% endif %}
normal table calls
```